tests/server/server-test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [ ! -d ./certs/ ]; then
+    #return 77 to indicate to automake that the test was skipped
+    exit 77
+fi
+
+# Skip test if filesystem disabled
+FILESYSTEM=`cat config.log | grep "disable\-filesystem"`
+if [ "$FILESYSTEM" != "" ]
+then
+    exit 77
+fi
+
+./wolfssl s_server -port 11111 -key ./certs/server-key.pem\
+    -cert ./certs/server-cert.pem -noVerify &
+pid_server=$!
+sleep 0.001
```

Comment:
fsantize github action test failed, possibly due to the server not starting up quick enough? If helpful you can use the 'ready_file' scheme for waiting and checking if the server is up. Something like adding a -readyFile=<file location> option to the s_server and leveraging the existing ready_file output code in the example server. Then waiting for that here in the test script for indicating if the server is listening on a port yet.